### DISCUSSION TITLE
单机部署无法使用问题

### DISCUSSION THREAD

**Discussion by zhangtao106** [apache/doris#34279] [A - General / Q&A]
按照官方文档中快速开始的步骤在centos7 虚拟机进行部署，操作完成后无法使用，be节点无法启动，Execution failed: Error Failed to execute sql: java.sql.SQLException: (conn=6) errCode = 2, detailMessage = There is no scanNode Backend available.[10167: not alive]，哪有有详细的部署文档或者操作手册之类的呢
![be](https://github.com/apache/doris/assets/42627122/a11cbd20-87b1-47bb-a1df-6019b5bd38e7)
![be-out](https://github.com/apache/doris/assets/42627122/e4ee8a79-d6d1-4d04-bcb9-d373daae7a75)

**Comment by yagagagaga** [comment]
You can try using docker or Doris Manager to deploy a Doris instance.
